public/locales/es.json

```diff
@@ -136,7 +145,7 @@
     "length": "24 marzo 2023",
     "type": "Venue",
     "description": "Mantente en contacto a través de nuestros canales (Discord, LinkedIn) para saber qué sede nos acogerá para este evento :)",
-    "heading": "¿Quieres saber más sobre la sede de este evento? :) Mantente informado en LinkedIn, Twitter o mejor aún, <link>Discord</link>!"
+    "heading": "Quiere saber más sobre el lugar :) ¡Manténgase actualizado en <linkedin>LinkedIn</linkedin>, <twitter>Twitter</twitter> o mejor aún, <discord>Discord</discord>!"
```

Comment:
Is this the cause of everything?

Reply:
This one was an actual error, in addition of the others "undefined missing translation" message